cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -372,6 +375,17 @@ pub mod pallet {
 	/// Whether or not the XCMP queue is suspended from executing incoming XCMs or not.
 	#[pallet::storage]
 	pub(super) type QueueSuspended<T: Config> = StorageValue<_, bool, ValueQuery>;
+
+	/// Initialization value for the DeliveryFee factor.
+	#[pallet::type_value]
```

Comment:
@wentelteefje this is also one of those weird storage related macros that you can shed some light in during #1714.